library/ssl_tls.c

```diff
@@ -6086,7 +6086,8 @@ void mbedtls_ssl_handshake_free( mbedtls_ssl_context *ssl )
                 }
                 else
                 {
-                    MBEDTLS_SSL_DEBUG_MSG( 2, ( "Reallocating in_buf to %d", buf_len ) );
+                    MBEDTLS_SSL_DEBUG_MSG( 2, ( "Reallocating in_buf to %lu",
+                                                (unsigned long) buf_len ) );
```

Comment:
[`buf_len` should have been a `size_t`](https://github.com/ARMmbed/mbedtls/pull/3809#discussion_r513060447). I think here it would be better to use `%zu` and cast to `size_t`. When we rework #3809, we'll remove the cast.

Same with the out case below.